src/connectors/youtube.js

```diff
@@ -203,6 +219,18 @@ async function fetchCategoryName(videoId) {
  * Asynchronously read connector options.
  */
 async function readConnectorOptions() {
+	if (await Util.getOption('YouTube', 'scrobbleIndividualTracks')) {
+		scrobbleIndividualTracks = true;
+		trackInfoGetters = scrobbleIndividualTracks ? [
```

Comment:
So this option just allows to reorder the list of getters. Do we really need this? Have you an example where both the video description and chapters contain track info?

Reply:
> So this option just allows to reorder the list of getters. Do we really need this? Have you an example where both the video description and chapters contain track info?

The idea of this change is to check chapters for track info *before* the title so that the individual tracks would be scrobbled instead of the whole album as a single track.

Concerning your second question: chapters are usually used for track info on album videos, and (according to [this YouTube support page](https://support.google.com/youtube/answer/9884579?hl=en)) they are generated from the description.  That's why I don't think that the description needs to be checked at all when chapters are present.

Reply:
You're right. chapters should be checked first. The only thing I don't understand why to make this behavior optional.